java/daikon/PptName.java

```diff
@@ -302,11 +302,40 @@ public boolean isExitPoint() {
    **/
   /*@EnsuresNonNullIf(result=true, expression="point")*/
   /*@Pure*/
+  public boolean isThrowPoint() {
+    return (point != null) && point.startsWith(FileIO.throw_suffix);
+  }
+
+  /**
+   * @return true iff this name refers to an abrupt completion point
+   **/
+  /*@EnsuresNonNullIf(result=true, expression="point")*/
+  /*@Pure*/
   public boolean isThrowsPoint() {
     return (point != null) && point.startsWith(FileIO.throws_suffix);
   }
 
   /**
+   * @return true iff this name refers to a procedure exception point
```

Comment:
The lack of parallelism in the javadoc ("abrupt completion point" versus "exception point") makes a reader wonder what specific distinction you are trying to highlight.  If there isn't one, then use the same term to prevent confusion.